flink-table/flink-table-common/src/main/java/org/apache/flink/table/plan/stats/ColumnStats.java

```diff
@@ -115,20 +189,81 @@ public String toString() {
 		if (max != null) {
 			columnStats.add("max=" + max);
 		}
+		if (maxValue != null) {
+			columnStats.add("max=" + maxValue);
+		}
 		if (min != null) {
 			columnStats.add("min=" + min);
 		}
+		if (minValue != null) {
+			columnStats.add("min=" + minValue);
+		}
 		String columnStatsStr = String.join(", ", columnStats);
 		return "ColumnStats(" + columnStatsStr + ")";
 	}
 
 	/**
 	 * Create a deep copy of "this" instance.
+	 *
 	 * @return a deep copy
 	 */
 	public ColumnStats copy() {
-		return new ColumnStats(this.ndv, this.nullCount, this.avgLen, this.maxLen,
-			this.max, this.min);
+		if (maxValue != null) {
```

Comment:
what if user didn't set `maxValue` but set `minValue`?

Reply:
`minValue != null` missing, full condition should be `maxValue != null || minValue != null`